src/wamr/wamr.cc

```diff
@@ -261,9 +261,11 @@ bool Wamr::link(std::string_view /*debug_name*/) {
     const wasm_name_t *name_ptr = wasm_importtype_name(import_types.get()->data[i]);
     const wasm_externtype_t *extern_type = wasm_importtype_type(import_types.get()->data[i]);
 
-    std::string_view module_name(module_name_ptr->data, module_name_ptr->size);
-    std::string_view name(name_ptr->data, name_ptr->size);
-    assert(name_ptr->size > 0);
+    assert(std::strlen(name_ptr->data) > 0);
```

Comment:
I know this is not new in this PR, but could you replace this `assert` with `fail(FailState::UnableToInitializeCode, ...)` like in other places? We shouldn't be crashing on invalid input.